pyews/core.py

```diff
@@ -44,15 +44,32 @@ def invoke(self, soap_body):
                 auth=(self.userconfiguration.credentials.email_address, self.userconfiguration.credentials.password),
                 verify=True
             )
+
+            __LOGGER__.debug('Response HTTP status code: %s', response.status_code)
+            __LOGGER__.debug('Response text: %s', response.text)
+
             parsed_response = BeautifulSoup(response.content, 'xml')
-            try:
-                if parsed_response.find('ResponseCode').string == 'ErrorAccessDenied':
-                    __LOGGER__.info('{}'.format(parsed_response.find('MessageText').string))
-                if parsed_response.find('ResponseCode').string == 'NoError':
-                    return parsed_response
-            except:
-                if parsed_response.find('ErrorCode').string == 'NoError':
-                    return parsed_response
+            if not parsed_response.contents:
+                __LOGGER__.warning(
+                    'The server responded with empty content to POST-request '
+                    'from {current}'.format(current=self.__class__.__name__))
+                return
```

Comment:
Just looking for anything in content of response. I encountered a case when EWS returned nothing, empty `reponse.content`, lol. So we don't bother with it latter.